UPGRADE.md

```diff
@@ -10,7 +10,10 @@ Follow instructions in a [monorepo upgrade guide](./docs/contributing/UPGRADE.md
 
 ### You are developing a project based on project-base
 * upgrade only your composer dependencies and follow instructions
-* if you want update your project with the changes from [shopsys/project-base], you can follow the *(optional)* instructions or cherry-pick from the repository whatever is relevant for you but we do not recommend rebasing or merging everything because the changes might not be compatible with your project as it probably evolves in time
+* if you want update your project with the changes from [shopsys/project-base],
```

Comment:
please, do not do such changes if not necessary - the diff is then slightly confusing :slightly_smiling_face: 

The longer lines might be hard to read in PHP Storm when you have preview and navigation windows open as well - for me, [soft wrapping](https://stackoverflow.com/questions/10351608/word-wrapping-in-phpstorm) is helpful from time to time :wink: